### PR TITLE
[Python] Operations pages batch 3 [Replace C# samples]

### PR DESCRIPTION
**Pull request by reebhub**
related issues:
[RDoc-2816](https://issues.hibernatingrhinos.com/issue/RDoc-2816/Python-Client-API-Operations-Maintenance-Indexes-Set-index-priority-Replace-C-samples) Set index priority
[RDoc-2817](https://issues.hibernatingrhinos.com/issue/RDoc-2817/Python-Client-API-Operations-Maintenance-Indexes-Get-index-errors-Replace-C-samples) Get index errors
[RDoc-2818](https://issues.hibernatingrhinos.com/issue/RDoc-2818/Python-Client-API-Operations-Maintenance-Indexes-Get-index-names-Replace-C-samples) Get index names
[RDoc-2819](https://issues.hibernatingrhinos.com/issue/RDoc-2819/Python-Client-API-Operations-Maintenance-Indexes-Get-index-Replace-C-samples) Get index
[RDoc-2820](https://issues.hibernatingrhinos.com/issue/RDoc-2820/Python-Client-API-Operations-Maintenance-Indexes-Get-indexes-Replace-C-samples) Get indexes
[RDoc-2821](https://issues.hibernatingrhinos.com/issue/RDoc-2821/Python-Client-API-Operations-Maintenance-Indexes-Get-terms-Replace-C-samples) Get terms
[RDoc-2822](https://issues.hibernatingrhinos.com/issue/RDoc-2822/Python-Client-API-Operations-Maintenance-Indexes-Index-has-changed-Replace-C-samples) Index has changed
[RDoc-2799](https://issues.hibernatingrhinos.com/issue/RDoc-2799/Python-Client-API-Operations-Server-Wide-Add-database-node-Replace-C-samples) Add database node
[RDoc-2800](https://issues.hibernatingrhinos.com/issue/RDoc-2800/Python-Client-API-Operations-Server-Wide-Compact-database-Replace-C-samples) Compact database
[RDoc-2801](https://issues.hibernatingrhinos.com/issue/RDoc-2801/Python-Client-API-Operations-Server-Wide-Toggle-database-state-Replace-C-samples) Toggle database state
[RDoc-2802](https://issues.hibernatingrhinos.com/issue/RDoc-2802/Python-Client-API-Operations-How-to-Switch-operation-to-diff-database-Replace-C-samples) Switch operation to diff database

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-index-errors.python.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+{PANEL/}
+
+{PANEL: Syntax}
+
+{CODE:python syntax_1@ClientApi\Operations\Maintenance\Indexes\GetIndexErrors.py /}
+
+| Parameters | Type | Description |
+| - | - | - |
+| **index_names** | `str` | List of index names to get errors for |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-index-errors.python.markdown:40]:
```suggestion
| **index_names** | `*str` | List of index names to get errors for |
```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+{PANEL: Syntax}
+
+{CODE:python get_index_terms_syntax@ClientApi\Operations\Maintenance\Indexes\GetIndexTerms.py /}
+
+| Parameters | Type | Description |
+| - | - | - |
+| **index_name** | `str` | Name of an index to get terms for |
+| **field** | `str` | Name of index-field to get terms for |
+| **from_value** | `str` (optional) | The starting term from which to return results.<br>This term is not included in the results.<br>`null` - start from first term. |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown:31]:
```suggestion
| **from_value** | `str` (optional) | The starting term from which to return results.<br>This term is not included in the results.<br>`None` - start from first term. |
```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| **page_size** | `int` | Number of terms to get.<br>`null` - return all terms.  |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown:32]:
```suggestion
| **page_size** | `int` | Number of terms to get.<br>`None` - return all terms.  |
```

**Review by poissoncorp — APPROVED** [review]

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-index-errors.python.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-index-errors.python.markdown:40]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown:31]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/operations/maintenance/indexes/get-terms.python.markdown:32]: done

**Pull request merged**